mail/flowmailer/src/test/kotlin/com/ritense/mail/flowmailer/FlowmailerTestConfiguration.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.mail.flowmailer
+
+import com.ritense.mail.flowmailer.autoconfigure.FlowmailerAutoConfiguration
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.test.context.TestConfiguration
+
+@SpringBootApplication(scanBasePackageClasses = [FlowmailerAutoConfiguration::class])
```

Comment:
Scan base packages zou niet hoeven denk ik dat gaat al automatisch is mijn stelling. Dat loopt via de META-INF folder.